### ISSUE TITLE
waitForViewWithAccessibilityLabel: fails on iOS7 if the label contains a newline

### ISSUE THREAD

**Issue by CRedit360** [kif-framework/KIF#265]
Our app has a load of labels with text like "No results found.\nTap to retry." and tests to verify that they're shown.  The tests pass on iOS6, but fail on iOS7.

I've added a test case at https://github.com/percysnoodle/KIF/tree/many_lines but I don't have a fix.  I'd like to see what the accessibility labels are (maybe iOS7 strips the newlines?) but the Accessibility Inspector is broken. (see http://openradar.appspot.com/radar?id=5720630150299648)
